### ISSUE TITLE
LC0003 fails on "Record Link"

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#42]
Seems to be a false report.

`RecordLink: Record "Record Link";`

![image](https://user-images.githubusercontent.com/25268332/134907082-0bf748f2-e93e-445f-ac83-314f6e228f57.png)

**Comment by fvet** [comment]
Most warnings on the variables have dissappeared since the last update / release. However, I'm still left with a similar one on function parameters.

_Do not use an Object ID for properties or variables declaration. Use Link instead._

![image](https://user-images.githubusercontent.com/25268332/134936140-2d7919d9-4f00-4907-88f3-c9f847b2c8b2.png)

```
[EventSubscriber(ObjectType::Codeunit, Codeunit::"Job Queue - Send Notification", 'OnBeforeRun', '', false, false)]
    local procedure OnBeforeJobQueueSendNotification(var JobQueueEntry: Record "Job Queue Entry"; var RecordLink: Record "Record Link"; var IsHandled: Boolean)

```

**Comment by StefanMaron** [comment]
Should be fixed in v0.14
